eclass/verify-sig.eclass

```diff
@@ -144,8 +145,9 @@ verify-sig_verify_detached() {
 			# gpg can't handle very long TMPDIR
 			# https://bugs.gentoo.org/854492
 			local -x TMPDIR=/tmp
-			gemato gpg-wrap -K "${key}" "${extra_args[@]}" -- \
-				gpg --verify "${sig}" "${file}" ||
+			gemato openpgp-verify-detached -K "${key}" \
```

Comment:
I wondered at first if we should let ebuilds require all good or a list of FPs which must be there, given it's kind of a nuisance to sift keys out of a keyring, but it's too much work and no real benefit. It's not even much more convenient.

So, looks good. Thank you for this!